### PR TITLE
Flux comms plan bootstrap

### PR DESCRIPTION
**Pull request by dholbach**
There are still quite a few things that need to be worked out, but I feel this should be a good start.

As this process is meant to serve @fluxcd/maintainers as well as its community, please speak up and LMK if this makes sense.

Also CC @staceypotter

### CODE REVIEW

**Comment by phillebaba** [comment]
Regarding the monthly blog, is there a plan on how news is going to be collected? Should each project send updates if there is any notable news for that month?

**Comment by stefanprodan** [comment]
@phillebaba one source of news is the meeting notes doc, another one is the changelog of each controller repo.

**Comment by dholbach** [comment]
My idea was that all maintainers (and or interested developers) feed items into the spreadsheet as noted in the doc. If you have time to write up a paragraph or two, you can also add it straight to the monthly post.

I think it'd be good if members of the comms team don't have to read (and make sense of) all changelogs.

**Comment by phillebaba** [comment]
@dholbach that was what I was trying to get at. As we grow it would mean more and more work for the communication team to aggregate all the information if they were responsible for gathering it.

**Comment by dholbach** [comment]
Yes. So the idea with the process described in this doc is:

- for maintainers/developers: bookmark the spreadsheet once, add important stuff, done, feel good about letting people know
- for comms folks: focus on getting the word out

Of course there can always be overlap between the two groups. :bouquet:

File: COMMUNITY.md [hunk 1]
```diff
@@ -13,13 +13,40 @@ As opposed to defining a big overarching burden of responsibilities the team wou
...
+We as a project want to make it a priority to inform our community and the wider world of developments well. It's what helps new contributors get up to speed, enables collaboration, integrations and more.
+
+Part of our team wants to own comms on
+
+- Slack: [#flux](https://cloud-native.slack.com/archives/flux), [#gitops](https://kubernetes.slack.com/archives/gitops), likely more to follow
+- the `cncf-flux-dev` [project mailing list](https://lists.cncf.io/g/cncf-flux-dev)
+- [our blog](https://github.com/fluxcd/website/issues/27) (WIP)
+- Twitter (WIP)
+- [LinkedIn](https://www.linkedin.com/groups/8985374/)
+- [Kubernetes Discuss - Ecosystem forum](https://discuss.kubernetes.io/c/announcements/announcements-eco)
```
Comment (hiddeco) [inline COMMUNITY.md:27]: The list looks a bit "messy" due to some items starting with a capital letter, and others don't. Maybe this is a bit better?

```suggestion
- The `cncf-flux-dev` [project mailing list](https://lists.cncf.io/g/cncf-flux-dev)
- [Our blog](https://github.com/fluxcd/website/issues/27) (WIP)
- Slack (likely more to follow)
  + [#flux](https://cloud-native.slack.com/archives/flux)
  + [#gitops](https://kubernetes.slack.com/archives/gitops)
- Social media
  + [LinkedIn](https://www.linkedin.com/groups/8985374/)
  + Twitter (WIP)
- [Kubernetes Discuss: Ecosystem forum](https://discuss.kubernetes.io/c/announcements/announcements-eco)
```

**Comment by dholbach** [comment]
@staceypotter Any feedback from you?

**Review by hiddeco — APPROVED** [review]
Not on the team, but given Scott is absent, hereby my approval. :dolphin:

File: COMMUNITY.md [hunk 2]
```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
...
 
-| Name           | Slack handle  | Github handle | Affiliation |
-| -------------- | ------------- | ------------- | ----------- |
-| Daniel Holbach | `@dholbach`   | `@dholbach`   | Weaveworks  |
-| Scott Rigby    | `@scottrigby` | `@scottrigby` | Weaveworks  |
+| Name           | Slack handle     | Github handle   | Affiliation |
+| -------------- | ---------------- | --------------- | ----------- |
+| Daniel Holbach | `@dholbach`      | `@dholbach`     | Weaveworks  |
+| Scott Rigby    | `@scottrigby`    | `@scottrigby`   | Weaveworks  |
+| Stacey Potter  | `@Stacey Potter` | `@staceypotter` | Weaveworks  |
```
Comment (scottrigby) [inline COMMUNITY.md:53]: Small syntax suggestion. For tables I usually just us a single space for cell padding – regardless of content width – and don't bother trying to justify it all. What this does is allow git diffs to only recognize changed lines, making diffs much easier to read future changes 👓 😸 

```suggestion
| Name | Slack handle | Github handle | Affiliation |
| -- | -- | -- | -- |
| Daniel Holbach | `@dholbach` | `@dholbach` | Weaveworks  |
| Scott Rigby | `@scottrigby` | `@scottrigby` | Weaveworks  |
| Stacey Potter | `@Stacey Potter` | `@staceypotter` | Weaveworks  |
```

File: COMMUNITY.md [hunk 3]
```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
   - Slack
 - Oversight of [this repository](https://github.com/fluxcd/community).
 
+### Communications
+
+We as a project want to make it a priority to inform our community and the wider world of developments well. It's what helps new contributors get up to speed, enables collaboration, integrations and more.
```
Comment (scottrigby) [inline COMMUNITY.md:18]: I'd like to suggest we adopt one sentence per line for markdown files, per the argument at https://sembr.org/. Not a requirement, just a suggestion. I had applied this for the GOVERNANCE and SECURITY md files, so maybe we can adopt that standard for this community repo, and discuss separately whether or not to adopt this standard across other fluxcd repos?

```suggestion
We as a project want to make it a priority to inform our community and the wider world of developments well.
It's what helps new contributors get up to speed, enables collaboration, integrations and more.
```

File: COMMUNITY.md [hunk 4]
```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
...
+  + Twitter (WIP)
+- [Kubernetes Discuss: Ecosystem forum](https://discuss.kubernetes.io/c/announcements/announcements-eco)
+
+This means that the team will figure out what kind of news should go where (process still TBD) and execute in a timely fashion.
+
+### How to use this
+
+#### As a maintainer or contributor
+
+Add news-worthy items to our [planning spreadsheet](https://docs.google.com/spreadsheets/d/1hPV3qJ95I_RKPyeo3zUJOPrHc0LNeSlP3f9fjXTen-c/edit#gid=0) - ask for editing permissions if necessary. Ping any of the folks at the bottom of this page, if urgent.
```
Comment (scottrigby) [inline COMMUNITY.md:38]: See one sentence per line suggestion above.

```suggestion
Add news-worthy items to our [planning spreadsheet](https://docs.google.com/spreadsheets/d/1hPV3qJ95I_RKPyeo3zUJOPrHc0LNeSlP3f9fjXTen-c/edit#gid=0) - ask for editing permissions if necessary.
Ping any of the folks at the bottom of this page, if urgent.
```

File: COMMUNITY.md [hunk 5]
```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
...
+
+#### As comms team member
+
+1. Check the [comms planning spreadsheet](https://docs.google.com/spreadsheets/d/1hPV3qJ95I_RKPyeo3zUJOPrHc0LNeSlP3f9fjXTen-c/edit#gid=0) and see if there are items that still need doing.
+1. We would like to get monthly updates out to our community, so writing up the post, getting quotes from relevant people, etc is of importance to us. Check the ["Monthly Updates" folder](https://drive.google.com/drive/u/0/folders/1-vODozxyaDruofIWfnrY8hFr2xauFatf) and please help getting things ready for the end of the month.
```
Comment (scottrigby) [inline COMMUNITY.md:43]: See one sentence per line suggestion above.

```suggestion
1. We would like to get monthly updates out to our community, so writing up the post, getting quotes from relevant people, etc is of importance to us.
    Check the ["Monthly Updates" folder](https://drive.google.com/drive/u/0/folders/1-vODozxyaDruofIWfnrY8hFr2xauFatf) and please help getting things ready for the end of the month.
```

File: COMMUNITY.md [hunk 6]
```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
...
-| -------------- | ------------- | ------------- | ----------- |
-| Daniel Holbach | `@dholbach`   | `@dholbach`   | Weaveworks  |
-| Scott Rigby    | `@scottrigby` | `@scottrigby` | Weaveworks  |
+| Name           | Slack handle     | Github handle   | Affiliation |
+| -------------- | ---------------- | --------------- | ----------- |
+| Daniel Holbach | `@dholbach`      | `@dholbach`     | Weaveworks  |
+| Scott Rigby    | `@scottrigby`    | `@scottrigby`   | Weaveworks  |
+| Stacey Potter  | `@Stacey Potter` | `@staceypotter` | Weaveworks  |
 
 Talk to us in [`#flux`](https://cloud-native.slack.com/archives/flux) on [Slack](https://slack.cncf.io) and let us know if you're interested in contributing. We would love to have you part of the team.
```
Comment (scottrigby) [inline COMMUNITY.md:55]: See one sentence per line suggestion above.

```suggestion
Talk to us in [`#flux`](https://cloud-native.slack.com/archives/flux) on [Slack](https://slack.cncf.io) and let us know if you're interested in contributing.
We would love to have you part of the team.
```

**Review by scottrigby — APPROVED** [review]
Coming back up for air. Overall accept/approve 👍  Just small syntax suggestions for better docs maintainability.

File: COMMUNITY.md (hunk 2 above)
Comment (dholbach) [reply COMMUNITY.md:53]: :shrug:

**Comment by dholbach** [comment]
Thanks a lot for the reviews everyone!

**Pull request merged**